docuseal/docker-compose.yml

```
@@ -0,0 +1,28 @@
+services:
+  app_proxy:
+    environment:
+      APP_HOST: docuseal_server_1
+      APP_PORT: 3000
+  server:
```

Add new line for better structure.
```suggestion
      APP_PORT: 3000

  server:
```